cli/sign.go

```diff
@@ -45,7 +45,7 @@ func signExisting(c *cli.Context) error {
 	if len(c.String("output-path")) > 0 {
 		outputFile = c.String("output-path")
 	}
-
+	artFileStat, _ := os.Stat(artFile)
```

Comment:
There are two things going on here, as far as I can tell:

1. It's usually always a good idea to always check the error code. 
2. Do the operation close to where it is used.